### DISCUSSION TITLE
UUID snippet not working anymore?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#3275] [Q&A] [answered]
There use to be a uuid snippet that would create a random uuid. The snippet seems to be there but  it doenst do anything anymore.

![image](https://github.com/LazyVim/LazyVim/assets/8405459/65d72ab4-0d1c-44da-b830-e53e95799ce3)

![image](https://github.com/LazyVim/LazyVim/assets/8405459/93ceaae6-8938-4e19-996c-a7c05979eaa9)

**Comment by dpetka2001** [comment]
That's probably a problem with the `nvim-snippets` cmp source. When Luasnip is enabled I get the same snippet but it correctly expands to a random uuid. Maybe create an issue at the `nvim-snippets`, so they can look into it?

**Reply by dpetka2001** [reply, marked as answer]
Actually there is an open [issue](https://github.com/garymjr/nvim-snippets/issues/11) that seems related and from the contents of it, it seems that this is something that the current `vim.snippet.expand` doesn't support because of LSP specification.
